lighthouse-cli/index.js

```diff
@@ -32,41 +32,62 @@ if (semver.lt(process.version, '5.0.0')) {
 
 const formatOptions = Object.values(Printer.OUTPUT_MODE).join(', ');
 
-const cli = meow(`
-Usage:
-    lighthouse [url]
-
-Basic:
-    --help             Show this help
-    --version          Current version of package
-
-Logging:
-    --verbose          Displays verbose logging
-    --quiet            Displays no progress or debug logs
-
-Run Configuration:
-    --mobile           Emulates a Nexus 5X (default=true)
-    --load-page        Loads the page (default=true)
-    --save-assets      Save the trace contents & screenshots to disk
-    --save-artifacts   Save all gathered artifacts to disk
-    --audit-whitelist  Comma separated list of audits to run (default=all)
-    --list-all-audits  Prints a list of all available audits and exits
-    --config-path      The path to the config JSON.
-
-Output:
-    --output           Reporter for the results
-                       Reporter options: ${formatOptions}  (default=pretty)
-    --output-path      The file path to output the results (default=stdout)
-                       Example: --output-path=./lighthouse-results.html
-`, {
-  // These options do not have a value
-  boolean: [
-    'save-assets', 'save-artifacts', 'list-all-audits',
-    'verbose', 'quiet', 'help', 'version'
-  ]
-});
-
-if (cli.flags.listAllAudits) {
+const cliOptions = {
```

Comment:
can you collapse this and the `forEach`/`cliGroup` calls into a direct chain of yargs calls? More verbose, but we don't have very many options so it shouldn't be too bad (and nice to have it all laid out directly)